src/Evolution/DgSubcell/Matrices.cpp

```diff
@@ -229,13 +229,13 @@ Matrix reconstruction_matrix_cache_impl_helper(
   // We use rhs_recons_matrix here as a temp buffer, we will fill it later.
   Matrix rhs_recons_matrix(num_pts + 1, num_subcells);
   dgemm_<true>('T', 'N', proj_matrix.columns(), proj_matrix.columns(),
-               proj_matrix.rows(), 2.0, proj_matrix.data(), proj_matrix.rows(),
-               proj_matrix.data(), proj_matrix.rows(), 0.0,
-               rhs_recons_matrix.data(), proj_matrix.columns());
+               proj_matrix.rows(), 2.0, proj_matrix.data(),
+               proj_matrix.spacing(), proj_matrix.data(), proj_matrix.spacing(),
+               0.0, rhs_recons_matrix.data(), rhs_recons_matrix.spacing());
 
   for (size_t l = 0; l < num_pts; ++l) {
     for (size_t j = 0; j < num_pts; ++j) {
-      lhs_recons_matrix(l, j) = *(rhs_recons_matrix.data() + (j + l * num_pts));
+      lhs_recons_matrix(l, j) = rhs_recons_matrix(j, l);
```

Comment:
Alternatives, if preferred:

- Add spacing to the pointer arithmetic
- Use `blaze::submatrix` and `blaze::trans` instead of the loops

Reply:
This should be fine, I think